ui/raidboss/data/06-ew/dungeon/aloalo_island.ts

```diff
@@ -137,19 +117,6 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { id: '8B95', source: 'Quaqua', capture: false },
       response: Responses.getBehind(),
     },
-    {
-      id: 'Aloalo Quaqua Howl',
-      type: 'StartsUsing',
-      netRegex: { id: '8B96', source: 'Quaqua', capture: false },
-      infoText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Adds',
```

Comment:
Usually "Adds" is when there are adds to kill. I think in this case, there's obvious ground markers to avoid, and so there's no need to tell players about adds.